packages/terra-status-view/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ Unreleased
 ----------
 ### Changed
 * Updated "test:jest" script to work on windows.
+* Fixed text and glyphs being able to overflow out of the Status View
```

Comment:
Can you add the heading `### Fixed` above this change? We follow [this guideline](https://keepachangelog.com/en/1.0.0/) to structure our changelogs so they are easier to see what occurred in each release. I just added this link to the doc we sent (I realized it wasn't on there!). 😄